.ci/kata-simplify-log.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+readonly script_name=${0##*/}
+
+die()
+{
+    local -r msg="$*"
+    echo >&2 "ERROR: $msg"
+    exit 1
+}
+
+usage()
+{
+    cat <<EOT
+Description: Simplify the specified logfile by replacing common fields with
+fixed strings to make diff(1)-ing easier.
+
+Usage: $script_name <log-file>
+       $script_name [-h|--help|help]
+
+Options:
+
+  -h     : Show this help.
+  --help :
+  help   :
+
+Limitations:
+
+- This script uses simple heuristics and might break at any time.
+
+EOT
+}
+
+simplify_log()
+{
+    local -r file="$1"
+
+    sed -r \
```

Comment:
Heh. heh heh - that may just be the most un-reviewable line we have had so far ;-)
If we go further with this script then maybe we can break this into an array with some comments and then iterate over it - or something in order to break this down and explain what the magic is doing :-)